app/src/androidTest/java/com/appunite/loudius/util/ScreenshotTestRule.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2023 AppUnite S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.appunite.loudius.util
+
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.base.DefaultFailureHandler
+import androidx.test.platform.app.InstrumentationRegistry.getInstrumentation
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+import java.util.concurrent.atomic.AtomicBoolean
+
+/**
+ * TestRule used to run all test methods try count 1 time. Take screenshots on failure.
```

Comment:
question: I don't understand why we need this retry count?

Reply:
Thanks to that we can set how many times we want to start test and take screenshot.

Reply:
I think you're right, I deleted it :)